gcc/rust/backend/rust-compile-pattern.cc

```diff
@@ -915,8 +951,26 @@ CompilePatternBindings::visit (HIR::SlicePattern &pattern)
     case TyTy::TypeKind::SLICE:
```

Comment:
Not sure if you actually need this case here but maybe just leave this rust_sorry_at for now and we will see